scripts/test_truffle.js

```diff
@@ -22,9 +22,17 @@ try {
     }
   });
 
-  exec('truffle.cmd test --network ganache', (err, stdout, stderr) => {
+  exec('truffle test --network ganache', (err, stdout, stderr) => {
```

Comment:
Should this be using the project's specified version of truffle? ie. `node_modules/.bin/truffle`?

Reply:
Yup, that's a good point.